pennylane_lightning_kokkos/src/simulator/StateVectorKokkos.hpp

```diff
@@ -903,30 +903,46 @@ template <class Precision> class StateVectorKokkos {
     }
 
     /**
-     * @brief Apply a PauliX operator to the state vector using a matrix
+     * @brief Templated method that applies special n-qubit gates.
      *
+     * @tparam Gate functor class for Kokkos dispatcher.
+     * @tparam Number of qubits.
```

Comment:
You forgot to add the variable names here.

Reply:
Done, thanks.